release-02/code/Makefile

```diff
@@ -0,0 +1,8 @@
+init:
```

Comment:
I think we do not need this file in our refactor.

Reply:
This is about the base code of master, I'll branch out again a create a new PR